File: 프로그래머스/unrated/181938. 두 수의 연산값 비교하기/두 수의 연산값 비교하기.py
def solution(a, b):
    answer = 0
    result_1 = str(a)+str(b)
    result_2 = 2*a*b
    
    if int(result_1) > result_2:
        answer = int(result_1)
    elif int(result_1) < result_2:
        answer = result_2
        
    return answer

File: 프로그래머스/unrated/181938. 두 수의 연산값 비교하기/두 수의 연산값 비교하기.java
class Solution {
    public int solution(int a, int b) {
        int answer = 0;
        
        String sum_1 = String.valueOf(a);
        String sum_2 = String.valueOf(b);
        String sum_result = sum_1 + sum_2;
        
        if (Integer.valueOf(sum_result) > 2 * a * b)
            return Integer.valueOf(sum_result);
        else if (Integer.valueOf(sum_result) < 2 * a * b)
            return 2 * a * b;
        else
            return Integer.valueOf(sum_result);
    }
}